mmv1/templates/terraform/custom_flatten/netapp_volume_replicaton_mirror_state.go.erb

```diff
@@ -0,0 +1,20 @@
+func flatten<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
+  // Actual state of replication_enabled depends on mirrorState. let's update it.
+  // This is to pickup manual user STOP/RESUME operations on the replication.
```

Comment:
I wonder if we want to set back `replication_enabled` based on `mirrorState` or set whatever it's specified